packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -100,10 +100,10 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
     );
 
     return (
-      <div {...divProps} className={classNames.coin}>
+      <div role="presentation" {...divProps} className={classNames.coin}>
         {// Render PersonaCoin if size is not size8. size10 and tiny need to removed after a deprecation cleanup.
         size !== PersonaSize.size8 && size !== PersonaSize.size10 && size !== PersonaSize.tiny ? (
-          <div {...divCoinProps} className={classNames.imageArea} style={coinSizeStyle}>
+          <div role="presentation" {...divCoinProps} className={classNames.imageArea} style={coinSizeStyle}>
```

Comment:
Do we need to set this for the PersonaPresence or Icon components as well in the tiny case?

Reply:
presence is already aria-hidden i believe. 

icons are automatically aria-hidden unless they have aria-labeled by. So i think we should be good.

Reply:
oh THIS isn't a good idea. If icon is an image it uses a div wrapper. That'll make problems if icons are in buttons. I can add that fix too

![image](https://user-images.githubusercontent.com/1434956/70828853-122b7700-1da1-11ea-921c-b2cc218a75ba.png)

Reply:
not 100% related to persona (doubt the icon would ever be an image), but i guess it's possible.

Reply:
@mtennoe i think i got all of the divs. Do you see any I missed? The only one without 'presentation' is the one that is already aria-hidden

Reply:
Cool! Looks good to me. I left one small comment with regards to the div->span change